powerquery-docs/Connectors/Anaplan.md

```diff
@@ -1,120 +1,201 @@
----
-title: Power Query Anaplan connector
-description: Provides basic information, prerequisites, and instructions on how to connect to your Anaplan data, as well as troubleshooting tips.
-author: cpopell
+# Anaplan Connector Power BI Desktop
 
-ms.topic: conceptual
-ms.date: 2/28/2022
-ms.author: dougklo
-ms.reviewer: kvivek
-LocalizationGroup: reference
----
+The Anaplan Connector for Power BI Desktop enables access to your Anaplan model exports. It also:
 
-# Anaplan
```

Comment:
This title change should not be revereted.

Reply:
agree